.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: "\U0001F41BBug report"
+about: Create a report to help us improve Optuna.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Bug
```

Comment:
How about skipping this title as the information is conveyed through the label (and it'll make the body a bit shorter)? (Ditto for the other temples.)

Reply:
Done: https://github.com/pfnet/optuna/pull/652/commits/6c724f65b6625bdd57447b5a90c77bd3b8b9aa2d